### ISSUE TITLE
Documentation Incorrect & Installation Path

### ISSUE THREAD

**Issue by jmount1992** [qcr/tools#26]
ReadMe talks about installation to `/usr/local/bin` but nothing is installed here. This isn't very clear.

Also with the new services tool, I think should reevaluate the installation path to be something like

```
<parent>/qcr/tools/qcr   < install location for qcr tools repo
<parent>/qcr/tools/<tool_name> < install location for other tools (e.g., services tool)
```

Obviously a problem with changing this would be breaking current users

**Comment by jmount1992** [comment]
Re-evaluated installation path. Found this solution. This repo gets cloned to `/var/tmp/qcr/` (existing method). However, any tools living in separate repos get cloned into `/var/tmp/qcr/installed-tools/<tool-name>` when they are invoked. The `installed-tools` folder is ignored by git. 

Still need to clear up documentation regarding the note discussing the install path for system configs tool. I believe this has something to do with Ansible and the install path chosen there. @DasGuna to provide comments around this.

**Comment by jmount1992** [comment]
Fixed with commit [337caf2](https://github.com/qcr/tools/commit/337caf2e89c27837855e7f238ba87a5371ae4855)
